core/src/services/gcs/backend.rs

```diff
@@ -354,6 +364,8 @@ impl Builder for GcsBuilder {
                 client,
                 signer,
                 token_loader,
+                token: self.config.token,
+                scope: self.config.scope,
```

Comment:
`scope` will be `DEFAULT_GCS_SCOPE` if not set.

Reply:
Thanks, I've altered this now.

I skimmed over that it was being set already with a default, I'll combine this with the direct `String` usage too. Much appreciated :+1: